rules/autoconsent/bigcommerce.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "bigcommerce",
+  "vendorUrl": "https://github.com/bigcommerce/consent-manager",
+  "prehideSelectors": [
+    "#cookies-banner-overlay",
+    "#consent-manager",
+    "div[data-consent-manager-dialog]"
+  ],
+  "detectCmp": [
+    {
+      "exists": "#consent-manager,div[data-consent-manager-dialog]"
+    }
+  ],
+  "detectPopup": [
+    {
+      "visible": "#consent-manager,div[data-consent-manager-dialog]",
+      "check": "any"
+    }
+  ],
+  "optIn": [
+    {
+      "if": { "exists": "div[data-consent-manager-dialog]" },
+      "then": [
+        {
+          "click": "div[data-consent-manager-dialog] form button[type='submit']"
+        }
+      ],
+      "else": []
+    },
+    {
+      "if": { "exists": "#consent-manager" },
+      "then": [
+        {
+          "click": "#consent-manager div > button[class]:last-child"
+        }
+      ],
+      "else": []
+    }
+  ],
+  "optOut": [
+    {
+      "if": { "exists": "div[data-consent-manager-dialog]" },
+      "then": [
+        {
+          "click": "div[data-consent-manager-dialog] form button:not([type='submit'])"
```

Comment:
In my testing, pressing this button was not enough because it triggers another dialog. We should probably actually click the radio buttons in the dialog, rejecting the individual categories, and then click "Save"

Reply:
I made some changes by manually referencing the DOM but I wasn't able to get the popup on my side again. Thus, I put another conditional step after all to handle the options popup. If you're able to see it, can you try to test it on your side?